lib/ctws/version.rb

```diff
@@ -1,7 +1,10 @@
 module Ctws
   # Under development: BETA
 
-  VERSION = '0.2.3.beta'
+  VERSION = '0.2.4.beta'
```

Comment:
I think this should be an increment to the MINOR rather than a PATCH, as this PR is adding functionality in a backwards-compatible manner or it's solving a bug?

Reply:
I agree with @agustibr that it should be a MINOR increment because it adds functionality although the backwards compatibility is discutible. Anyway, as we're in beta we're still accomplishing with semver.